test/evergreen.yml

```diff
@@ -97,6 +97,19 @@ functions:
           set -o verbose
           echo '#undef HAVE_FTRUNCATE' >> wiredtiger_config.h
     - *make_wiredtiger
+  "compile wiredtiger sanitizer":
```

Comment:
To avoid confusion with other sanitizers e.g. memory, undefined behavior, suggest adding `address` into the function name.

Reply:
Will fix it.